egs2/vctk_noisy/enh1/conf/tuning/train_enh_dptfsnet.yaml

```diff
@@ -0,0 +1,59 @@
+optim: adam
+init: xavier_uniform
+max_epoch: 100
+batch_type: folded
+batch_size: 4 # batch_size 16 can be trained on 4 RTX 2080ti
+iterator_type: chunk
+chunk_length: 32000
+num_workers: 4
+optim_conf:
+    lr: 1.0e-03
+    eps: 1.0e-08
+    weight_decay: 0
+patience: 10
+val_scheduler_criterion:
+- valid
+- loss
+best_model_criterion:
+-   - valid
+    - si_snr
+    - max
+-   - valid
+    - loss
+    - min
+keep_nbest_models: 1
+scheduler: steplr
+scheduler_conf:
+    step_size: 2
+    gamma: 0.98
+
+encoder: conv
+encoder_conf:
+    channel: 64
+    kernel_size: 2
+    stride: 1
+decoder: conv
+decoder_conf:
+    channel: 64
+    kernel_size: 2
+    stride: 1
+separator: dptfsnet
+separator_conf:
+    num_spk: 1
+    layer: 6
```

Comment:
I tested locally on a single GTX 1080ti GPU. The input is of shape (2, 32000).
And I found that it could only fit into the GPU memory when layer=2.
It consumes around 10367 MiB GPU memory for the separator forward process (with gradient).
I am not sure whether the current config file can be used directly on a RTX 2080ti GPU (as stated).